.github/workflows/release-embedding-sdk.yml

```diff
@@ -186,8 +183,43 @@ jobs:
           name: metabase-sdk
           path: sdk
 
+      - name: Generate necessary files in the build directory
+        run: yarn run embedding-sdk:generate-package
+
+      - name: Upload built SDK package as artifact, so the artifact contains all the necessary files e.g. package.json, LICENSE.md, CHANGELOG.md
+        uses: actions/upload-artifact@v4
+        with:
+          name: final-metabase-sdk
+          path: ./resources/embedding-sdk
+
       - name: Publish to NPM
         run: |
           cd sdk
           echo //registry.npmjs.org/:_authToken=${{ secrets.NPM_RELEASE_TOKEN }} > .npmrc
-          npm publish
+          npm publish --dry-run
```

Comment:
This ensure we don't publish anything until we're sure this workflow runs like what we expected.